code/modules/research/experimentor.dm

```diff
@@ -78,21 +78,25 @@
 			if(initial(tempCheck.icon_state) != null)
 				critical_items += I
 
-
-/obj/machinery/r_n_d/experimentor/New()
-	..()
+/obj/machinery/r_n_d/experimentor/Initialize(mapload)
+	. = ..()
```

Comment:
```suggestion
	..()
```
Same here. All paths return `INITIALIZE_HINT_LATELOAD` thus keeping the return value of the parent isn't needed